### PR TITLE
Partial fix for issue 287. Still waiting for BOMs

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: kitchensink-jsp/pom.xml [hunk 1]
```diff
@@ -42,6 +42,14 @@
       <!-- 
       javaee6.with.tools.version>1.0.0.Final-redhat-1</javaee6.with.tools.version> 
       -->
+      <!-- Define the version of JBoss' Java EE 6 APIs and Tools we want
```
Comment (pmuir) [inline kitchensink-jsp/pom.xml:45]: I think you wanted to change this to the Hibernate line

Define the version of JBoss' Java EE 6 APIs, and of the Hibernate family BOM we want to import

File: payment-cdi-event/pom.xml [hunk 2]
```diff
@@ -34,6 +34,14 @@
       <!-- [WARNING] Using platform encoding (UTF-8 actually) to copy filtered 
          resources, i.e. build is platform dependent! -->
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <!-- Define the version of JBoss' Java EE 6 APIs and Tools we want 
+          to import. -->
```
Comment (pmuir) [inline payment-cdi-event/pom.xml:38]: I think you wanted to change this to the Hibernate line

Define the version of JBoss' Java EE 6 APIs, and of the Hibernate family BOM we want to import

Comment (sgilda) [inline payment-cdi-event/pom.xml:38]: Hi Pete,
I actually copied this from lines 38 - 45 in the kitchensink pom.xml 
file. Should it be changed there too?

**Comment by pmuir** [comment]
I added a commit that resolved some of the comment issues.

**Pull request closed without merging**